### DISCUSSION TITLE
How to unmap bufferline H and L

### DISCUSSION THREAD

**Discussion by Josh-Piper** [LazyVim/LazyVim#1162] [General]
Hi,

I was wondering how we could get the original `H` and `L` back, and be able to remap the bufferline change tab left and right to another binding, I have the below configuration but this won't bring back the original H and L. This should be correct based on the plugins part of the docs?

```
  "akinsho/bufferline.nvim",
  keys = {
    { "<S-h>", false },
    { "<S-l>", false },
  },
```

**Comment by dpetka2001** [comment]
That's because they're not mapped as `keys` of the plugin. They're declared in `keymaps.lua` with `vim.keymap.set` (see [here](https://github.com/LazyVim/LazyVim/blob/e797e13bcee980227e20c287c561a75d97f9a2e4/lua/lazyvim/config/keymaps.lua#L43C1-L52)). So if you want to change them, just change them with `vim.keymap.set` in your own `/lua/config/keymaps.lua` to whatever you want or use `vim.keymap.del` to disable them.
